fastlane/Fastfile

```diff
@@ -80,21 +82,39 @@ platform :ios do
 
   desc "Testflight build for Live Channel"
   lane :testflight_live do
+
+    udpated_version_number = increment_version_number(
+      xcodeproj: project,
+      version_number: File.read("VERSION")
+    )
+
+    udpated_build_number = increment_build_number(
+      xcodeproj: project
+    )
+
     match(
-      app_identifier: ['com.ecosia.ecosiaapp', 'com.ecosia.ecosiaapp.WidgetKit', 'com.ecosia.ecosiaapp.ShareTo'], 
+      app_identifier: ['com.ecosia.ecosiaapp', 'com.ecosia.ecosiaapp.WidgetKit', 'com.ecosia.ecosiaapp.ShareTo'],
       type: 'appstore',
       readonly: true
     )
+
     gym(
       scheme: "Ecosia",
       export_method: "app-store",
       configuration: "Release"
     )
+
     pilot(
       skip_waiting_for_build_processing: true, 
       apple_id: "670881887",
       itc_provider: "33YMRSYD2L"
     )
+
+    add_git_tag(
```

Comment:
I think we would also need to push the added tag to git, wouldn't we? Or does fastlane do it automatically?

Reply:
Bit rusty here 🤦 , I completely forgot that `add_git_tag` doesn't perform the git push itself! Thanks! Will add